### PR TITLE
174654244 Deformation tab

### PR DESCRIPTION
**Pull request by dstrawberrygirl**
This merges work on the model from @sfentress with some framework pieces to include the tab and a simple block to run the model.

This PR and subsequent PRs should probably remain separate from the current Seismic work in Master while that is nearing release

### CODE REVIEW

File: src/stores/stores.ts [hunk 1]
```diff
@@ -114,6 +116,7 @@ function getStoreSubstate(blocklyStoreProps: string[], tephraSimulationProps: st
       blocklyStore: pick(blocklyStoreProps)(blocklyStore),
       tephraSimulation: pick(tephraSimulationProps)(tephraSimulation),
       seismicSimulation: {},      // nothing to save yet
+      deformationSimulation: {},
```
Comment (sfentress) [inline src/stores/stores.ts:119]: I don't think this needs to be here. These are serializations of the various MST stores, and we don't have a `deformationSimulation` store yet.

File: src/stores/stores.ts [hunk 2]
```diff
@@ -145,7 +148,10 @@ export const deserializeState = (serializedState: SerializedState): IStoreish =>
   if (serializedState.version === 1) {
     return serializedState.state;
   }
-  return {unit: {name: "Tephra"}, blocklyStore: {}, tephraSimulation: {}, seismicSimulation: {}, uiStore: {}};
+  return {
+    unit: { name: "Tephra" },
+    blocklyStore: {}, tephraSimulation: {}, seismicSimulation: {}, deformationSimulation: {}, uiStore: {}
+  };
```
Comment (sfentress) [inline src/stores/stores.ts:154]: likewise.

**Review by sfentress — CHANGES_REQUESTED** [review]
This looks good, except for the question about the serialized state.

File: src/stores/stores.ts (hunk 1 above)
Comment (dstrawberrygirl) [reply src/stores/stores.ts:119]: Good catch - I had rolled back changes after our discussions earlier in the week and moved things to be part of the seismic simulation instead, but these managed to sneak past me.

**Review by sfentress — APPROVED** [review]
Looks good :+1:

**Pull request merged**